src/applications/vaos/containers/PreferredDatePage.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import FormButtons from '../components/FormButtons';
+import {
+  openFormPage,
+  updateFormData,
+  routeToNextAppointmentPage,
+  routeToPreviousAppointmentPage,
+} from '../actions/newAppointment.js';
+import { getPreferredDate } from '../utils/selectors';
+import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
+
+const initialSchema = {
+  type: 'object',
+  required: ['preferredDate'],
+  properties: {
+    preferredDate: {
+      type: 'string',
+    },
+  },
+};
+
+const uiSchema = {
+  preferredDate: {
+    'ui:title': "What is the earliest date you'd like to be seen?",
+    'ui:widget': 'date',
+    'ui:options': {
+      hideLabelText: true,
+    },
+    'ui:validations': [
+      (errors, preferredDate) => {
+        if (moment(preferredDate).isBefore(moment(), 'day')) {
```

Comment:
Weird that we don't already have a common validator for this.

Reply:
Yeah.. I actually couldn't find too many examples of us using this widget, and the documentation is a bit sparse.

Reply:
I think we generally use the `uiSchema` in `src/platform/forms-system/src/js/definitions/date.js`, there are more instances of that.